app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -23,8 +23,33 @@ class ReportbackController extends ApiController
     public function __construct(ReportbackService $reportbackService)
     {
         $this->reportbackService = $reportbackService;
-        $this->transformer = new ReportbackTransformer;
-        $this->itemTransformer = new ReportbackItemTransformer;
+        $this->transformer = new PhoenixGalleryTransformer;
```

Comment:
@sbsmith86 yeah I wasn't sure how to handle that since there already is a `ReportbackTransformer` but since that isn't needed anymore I could overwrite it with this?